framework/web/CHttpCookie.php

```diff
@@ -33,6 +33,14 @@ class CHttpCookie extends CComponent
 	 * @see $sameSite
 	 */
 	const SAME_SITE_STRICT = 'Strict';
+    /**
+     * SameSite policy None will allow all the cookie sent by the browser in all cross-site browsing context during
+     * all request methods (e.g. POST, PUT, PATCH etc).
+     * E.g. a POST request from https://otherdomain.com to https://yourdomain.com or a user following a link from
+     * https://otherdomain.com to https://yourdomain.com it will include the cookie.
+     * @see $sameSite
+     */
+    const SAME_SITE_NONE = 'None';
```

Comment:
Thank you for adding this :+1: . It was a mistake of mine not to add it when I added the `Lax` and `Strict` (ref: https://github.com/PeoplePerHour/yii/pull/11/files#diff-a3e61a4c64619ec82ac3a2cb7e0eb774R27)